include/chopper/layout/configuration.hpp

```diff
@@ -37,6 +44,32 @@ struct configuration
     bool force_all_binnings{false};
     bool output_statistics{false};
     bool debug{false};
+
+private:
+    friend class cereal::access;
+
+    template <typename archive_t>
+    void serialize(archive_t & ar)
+    {
+        uint32_t version{1};
+
+        ar(CEREAL_NVP(version));
+        ar(CEREAL_NVP(input_prefix));
+        ar(CEREAL_NVP(count_filename));
+        ar(CEREAL_NVP(sketch_directory));
+        ar(CEREAL_NVP(t_max));
+        // ar(CEREAL_NVP(aggregate_by_column));
+        ar(CEREAL_NVP(num_hash_functions));
+        ar(CEREAL_NVP(fp_rate));
+        ar(CEREAL_NVP(alpha));
+        ar(CEREAL_NVP(max_ratio));
+        ar(CEREAL_NVP(num_threads));
+        ar(CEREAL_NVP(estimate_union));
+        ar(CEREAL_NVP(rearrange_bins));
+        ar(CEREAL_NVP(determine_num_bins));
+        ar(CEREAL_NVP(force_all_binnings));
+        ar(CEREAL_NVP(output_statistics));
```

Comment:
Another point for discussion:

Would it be better to have the command line name here?
I think most of them are?

As an example:
If an option `alpha` is stored in member `a`, the JSON would be something like `a: 2`, while `alpha: 2` would be more helpful, because we clearly mean `--alpha 2`.

But as I said, I don't know if this is already the case :)

But even if, it might be a follow-up task to write a test that ensures that the names used in the command line and config are the same.

Reply:
Uh yes that is important. Then this PR will wait for #87 and then make sure that all the members are called the same.

Reply:
done